### ISSUE TITLE
PERFORMANCE NOT IMPROVED WITH RESNET152 BACKBONE

### ISSUE THREAD

**Issue by adityac8** [facebookresearch/maskrcnn-benchmark#832]
## 🐛 Bug

When I am changing the backbone to resnet152 there is not much performance improvement in respect to the resnet101 backbone. What more parameters are needed to be changed more?

**Comment by fmassa** [comment]
I'd recommend having a look at the config from Detectron https://github.com/facebookresearch/Detectron/blob/master/configs/12_2017_baselines/e2e_mask_rcnn_X-152-32x8d-FPN-IN5k_1.44x.yaml

They seem to use multi-scale training, and use a backbone pre-trained on ImageNet 5k.

**Comment by cskarthik7** [comment]
I get nan loss when I remove pretrained backbone resnet model, so how should I go about it?
